module/VuFind/src/VuFind/Resolver/Driver/PluginManager.php

```diff
@@ -61,13 +62,15 @@ class PluginManager extends \VuFind\ServiceManager\AbstractPluginManager
         'VuFind\Resolver\Driver\Threesixtylink' =>
             'VuFind\Resolver\Driver\DriverWithHttpClientFactory',
         'VuFind\Resolver\Driver\Demo' =>
-            'Zend\ServiceManager\Factory\InvokableFactory',
```

Comment:
The Demo driver has had a default value added to its constructor so that it does not require an OpenURL config setting to be passed in. I think it might make sense to either leave it as an invokable or make some different revisions, or else the new factory may throw unnecessary notices if there is no URL specified in the [OpenURL] configuration.